docs/source/notebooks/ModelBuilder in PyMC-Marketing context.ipynb

```diff
@@ -0,0 +1,1149 @@
+{
```

Comment:
<p>Now that we dropped 3.8 in PyMC, you can use dictionary update with <code>|</code>:</p><p><a href="https://docs.python.org/3/whatsnew/3.9.html#dictionary-merge-update-operators" target="_blank">https://docs.python.org/3/whatsnew/3.9.html#dictionary-merge-update-operators</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Is it possible to control the beta and alphas of beta distributions? In case the user needs to send additional information? + If so, this values should be scaled as well, right? or could be passed on real scale?</p><p> </p><p>Example:</p><pre class="ql-syntax" spellcheck="false">{'beta_channel': {'sigma': prior_sigma, 'alpha':prior_alphas, 'beta':prior_betas, 'dims': ('channel',)}}
</pre>

Reply:
<p>the DelayedSaturatedMMM doesn't implement beta prior, there's alpha, which priors can be controlled by changing their values in model_config.</p>